cmd/root.go

```diff
@@ -52,6 +53,33 @@ var libbeatConfigOverrides = []cfgfile.ConditionalOverride{{
 				"rotateeverybytes": 10 * 1024 * 1024,
 			},
 		},
+	}),
+}, {
+	Check: func(cfg *common.Config) bool {
+		if eflag := flag.CommandLine.Lookup("E"); eflag != nil {
```

Comment:
can we add a todo and an issue in libbeat to add support for this (maybe eg via a `Transform(*common.Config) *common.Config`) ?

Reply:
I'm going to revert this bit, but if I come up against it again then I will.